monitoring/prometheus/prometheus_staging.yml

```diff
@@ -0,0 +1,14 @@
+global:
+  scrape_interval: 30s
+  external_labels:
+    monitor: 'evalai-monitor'
+scrape_configs:
+  - job_name: 'prometheus' 
+    static_configs: 
+      - targets: ['https://staging.eval.ai:9090']
```

Comment:
Do you expect prometheus to also have `https://staging.eval.ai` as the domain name. This doesn't make sense. If this is the prometheus endpoint then you can use `localhost` here as only the backend servers will be on different instance

Reply:
@Ram81 I mentioned it in the above commentsweat_smile I have made change to take that file in the staging environment in PR 3447